README.md

```diff
@@ -1,6 +1,6 @@
 <div align="center">
   <img height="142" src="https://github.com/GooseFX1/gfx-web-app/blob/dev/public/img/assets/gfx_logo_gradient_lite.svg" />
-  <h3>GooseFX SDK Mono Repo</h3>
+  <h3>GooseFX Typescript SDK</h3>
```

Comment:
This repo contains not only typescript but also rust sdk

Reply:
Correct, the README at the root of the project says "GooseFX SDK Mono Repo" but this README file is in the `ts` directory so its only referring to the ts sdk.